app/components/Hero.tsx

```diff
@@ -0,0 +1,43 @@
+// LIBRARY IMPORTS
+import { Box, Heading } from "@chakra-ui/react";
+import Image from "next/image";
+
+// LOCAL IMPORTS
+import heroImage from "../../public/images/hero-photo.png";
+
+// HERO COMPONENT DEFINITION
+export default function Hero() {
+  return (
+    <Box
+      position="relative"
+      textAlign="center"
+      bg="white"
+      minH="calc(38.25vw + 6rem)"
+    >
+      <Image
+        src={heroImage}
+        alt={"machines photo"}
+        layout="responsive"
+        width={1728}
+        height={610}
+        objectFit="cover"
+        objectPosition="center"
+      />
+      <Box
+        position="absolute"
+        w="93%"
```

Comment:
From Figma, I see that the edges of the rectangle align with the contact info in the footer. In my footer component I used px="63px" for large screens and I wonder if that will line up with your rectangle or if I'll need to convert that padding to something dynamic like you have here. Or is it something somewhat minor that it's okay if it doesn't perfectly line up?

Reply:
Yeah I'm not positive on how to best translate layouts and ratios from the wireframe so its relatively similar on a standard screen size. I compared the fixed pixel width of the whole wireframe to the pixel width of this component and calculated 93% that way but we should definitely look into this more!